06.bowling_object/frame.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+class Frame
+  def initialize(shots, frame_number)
+    @shots = shots
+    @frame_number = frame_number
+  end
+
+  def calculate_score(frames)
+    next_frame = frames[@frame_number + 1]
+    after_next_frame = frames[@frame_number + 2]
+
+    if next_frame.nil?
+      points_sum
+    else
+      points_sum + bonus_points(next_frame, after_next_frame)
+    end
```

Comment:
```suggestion
    return points_sum if next_frame.nil?

    bonus_point =
      if strike?
        after_next_frame = frames[@frame_number + 2]
        strike_points(next_frame, after_next_frame)
      elsif spare?
        spare_points(next_frame)
      end

    points_sum + bonus_point.to_i
```

みたいにして `bonus_points` メソッドを無くすのも一手かも（好み）

Reply:
`bonus_points`メソッドを削除しました。